### PR TITLE
SLING-11158 use feature launcher assembly to launch the starter

### PR DESCRIPTION
**Pull request by enapps-enorman**
unpack the feature launcher assembly during the build and change the readme instructions

### CODE REVIEW

**Comment by tomaszmichalak** [comment]
Checked and it is working on my laptop.

File: pom.xml [hunk 1]
```diff
@@ -156,14 +156,11 @@
...
-                            <goal>get</goal>
-                            <goal>copy</goal>
+                            <goal>unpack</goal>
                         </goals>
                         <phase>package</phase>
                         <configuration>
-                            <!-- 1.1.28 and newer versions don't work out-of-the-box due to SLING-10956 -->
-                            <artifact>org.apache.sling:org.apache.sling.feature.launcher:1.1.26</artifact>
-                            <stripVersion>true</stripVersion>
+                            <artifact>org.apache.sling:org.apache.sling.feature.launcher:1.2.0:tar.gz</artifact>
```
Comment (rombert) [inline pom.xml:163]: Why did you remove the `stripVersion` config? With it configured we don't have to update the documentation every time we update the feature launcher version.

Comment (enapps-enorman) [reply pom.xml:163]: > Why did you remove the `stripVersion` config?

Mainly because the stripVersion config doesn't appear to be supported by the unpack goal.  That config was for renaming the artifact.  The version number you are seeing is the folder name from inside the .tar.gz file that was unpacked.  

I'll try to use the <fileMappers> config to rewrite the target path of that folder to achieve the same result.

**Comment by dalbani** [comment]
Now that https://github.com/apache/sling-feature-launcher-maven-plugin/pull/4 has been merged and the newly released plugin has been put into the POM with https://github.com/apache/sling-org-apache-sling-starter/commit/768a5537d4e0b89a9af7b021ac1d8bd2852259a8, I suppose this MR could be merged now?

**Comment by enapps-enorman** [comment]
> I suppose this MR could be merged now?

Yes, I'll do a final verification in the next day or two and merge it it all looks good with the latest code.
